lib/omniauth/strategies/mediawiki.rb

```diff
@@ -115,6 +124,10 @@ def authorize_url_parameters
         end
       end
 
+      def valid_request_token?(request_token)
+        request_token.callback_confirmed? && request_token.token && request_token.secret
```

Comment:
I'm not sure what `callback_confirmed?` is for. Are you sure this being truthy is required for a valid request?